General | Remove fake-hwclock, if real hwclock is available

MichaIng opened the pull request:
**Status**: Ready

**Commit list/description**:
- Apply on first run setup
- Apply on v6.13 patch
- Add changelog

_______________

Or the other way round, ship images without "fake-hwclock" and install, if no real hwclock found on first run setup? Could lead to 1970 time stamp until first time sync (?), not sure if this could be a problem.

MichaIng committed:
v6.13

+ DietPi-Software | 	Remove fake-hwclock, if real hwclock is available on first run setup

MichaIng committed:
v6.13

+ DietPi-Patchfile | Remove fake-hwclock, if real hwclock is available

MichaIng committed:
v6.13

+ CHANGELOG | Remove fake-hwclock, if real hwclock is available

Fourdee commented on dietpi/dietpi-software:
```diff
@@ -14295,6 +14295,9 @@ _EOF_
 		if (( $G_DIETPI_INSTALL_STAGE == 0 )); then
 
+			# - Remove fake-hwclock, if real hwclock is available
+			hwclock && G_AGP fake-hwclock
```
Minor, dev null this to prevent error print?
```
root@DietPi:~# hwclock && G_AGP fake-hwclock
hwclock: Cannot access the Hardware Clock via any known method.
hwclock: Use the --debug option to see the details of our search for an access method.
root@DietPi:~# hwclock &> /dev/null && G_AGP fake-hwclock
root@DietPi:~#
```

MichaIng replied:
@Fourdee 
Jep, good point. Thought a bid more verbosity about why and what we do on first run setup is not bad, but this is just too trivial 😉.

MichaIng committed:
Merge branch 'testing' into fake-hwclock

MichaIng committed:
v6.13

+ Minor conflict resolve fix

MichaIng committed:
v6.13

+ DietPi-Software | Suppress hwclock check output  + minor DietPi-Automation code and output improvements

Fourdee committed:
The one that got away!

😃

Fourdee commented:
@MichaIng 

Nice one, looking good 👍 

Merging now.

Fourdee commented:
> Could lead to 1970 time stamp until first time sync (?), not sure if this could be a problem.

Should be fine, sync is required early on in 1st run stage. Then after reboot, the hardware clock should take over.

Either way, i'll test on Native PC, will come back if any issues.
🈯️ Runs fine 👍

MichaIng commented:
@Fourdee 
This means we could switch it, install if needed instead of purge of not needed.
But I think it's good for now. Can be switched if there is really not report about wrong hwclock check e.g.
I will also test on VM with disabled hwclock.

Effect on image size is minimal anyway.
